src/components/Auth/Login/Login.react.js

```diff
@@ -192,9 +194,13 @@ class Login extends Component {
       }
       case 'password': {
         const password = event.target.value.trim();
+        const passwordLength = !(password.length >= 6);
```

Comment:
`!(password.length >= 6)` is not equal to `passwordLength`. Please name it suitably or change the conditions.

Reply:
@akshatnitd I have done the changes, sir. Please review it again. :)